### PR TITLE
GUI mappings, take two

### PR DESCRIPTION
**Pull request by asiekierka**
based on work by @shedaniel

### CODE REVIEW

File: mappings/net/minecraft/client/gui/CommandBlockScreen.mapping [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
 	FIELD C redstoneTriggerButton Lcrm;
-	FIELD D type Lbpi$a;
+	FIELD D mode Lbpi$a;
 	FIELD E conditional Z
 	FIELD F alwaysActive Z
+	FIELD z blockEntity Lbpi;
+	METHOD v updateMode ()V
+	METHOD w cycleType ()V
+	METHOD x updateConditionalMode ()V
+	METHOD y updateAlwaysActive ()V
```
Comment (shedaniel) [inline mappings/net/minecraft/client/gui/CommandBlockScreen.mapping]: Will `updateRedstoneMode` be better?

Comment (asiekierka) [reply mappings/net/minecraft/client/gui/CommandBlockScreen.mapping]: I think so, but would like a 2nd opinion

Comment (shedaniel) [reply mappings/net/minecraft/client/gui/CommandBlockScreen.mapping]: Not sure. Maybe `updateActivationMode`?

**Review by Gegy — APPROVED** [review]

File: mappings/net/minecraft/client/gui/CommandBlockScreen.mapping (hunk 1 above)
Comment (asiekierka) [reply mappings/net/minecraft/client/gui/CommandBlockScreen.mapping]: No, a 2nd opinion - as in an opinion from a person who is neither me nor you

Comment (shedaniel) [reply mappings/net/minecraft/client/gui/CommandBlockScreen.mapping]: Ahh

Comment (Gegy) [reply mappings/net/minecraft/client/gui/CommandBlockScreen.mapping]: `updateActivationMode` seems good

Comment (modmuss50) [reply mappings/net/minecraft/client/gui/CommandBlockScreen.mapping]: I agree `updateActivationMode` does seem good

Comment (asiekierka) [reply mappings/net/minecraft/client/gui/CommandBlockScreen.mapping]: Done.

**Comment by asiekierka** [comment]
Also fixes https://github.com/FabricMC/yarn/issues/476

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**